cmake/common/eprosima_libraries.cmake

```diff
@@ -12,45 +12,87 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Macro to find cmake built thirdparty libraries.
+#
+# Arguments:
+#   :package: The name of the packge to find. Used for find_package(${package})
+#   :extra arguments: Arguments passed after ${package} are parsed with cmake_parse_arguments()
+#
+# Related CMake options:
+#   :THIRDPARTY: Activate the use of internal thirdparties if external packges is not found [Defaults: OFF]
+#   :THIRDPARTY_FORCE: Force the use of internal thirdparties [Defaults: OFF]. Set to ON if EPROSIMA_BUILD is set to ON.
```

Comment:
Looking at the code below, I see that `EPROSIMA_BUILD` sets `THIRDPARTY` to ON, not `THIRDPARTY_FORCE`. Either change the behavior or the documentation.